servicemesh-release-notes/topics/rn-new-features.adoc

```diff
@@ -6,8 +6,11 @@ Reason – If known, include why has the enhancement been implemented (use case,
 Result – If changed, describe the current user experience
 ////
 
+= New features Technology Preview 9
+The release of {ProductName} adds support for Kiali 0.15.x, Jaeger 1.11, the 3scale Istio Adapter 0.4.1, and Istio 1.1.0-rc.2.
+
 = New features Technology Preview 8
-The release of {ProductName} adds support for Kiali 0.14.x and the 3scale Istio Adapter 0.3.  
+The release of {ProductName} adds support for Kiali 0.14.x and the 3scale Istio Adapter 0.3.
```

Comment:
Should be Kiali version 0.15.0 and 3Scale 0.4.1.

Reply:
@JStickler You're looking at the New Features for TP8. The next line up contains the new features for TP9.

Reply:
D'oh!  Never mind then!